site/src/components/keyboard-controls/KeyboardControls.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC, ReactNode } from "react";
+import clsx from "clsx";
+import tableStyles from "../mdx/table/Table.module.css";
+import styles from "./KeyboardControls.module.css";
+
+export interface KeyboardControlsProps {
+  children: ReactNode;
+  className?: string;
+}
+
+export const KeyboardControls: FC<KeyboardControlsProps> = ({
+  children,
+  className,
+}) => {
+  return (
+    <table className={clsx(styles.table, tableStyles.table, className)}>
```

Comment:
Might be best to import the site's mdx table (`import { Table } from "../mdx/table";`) to preserve styling (the Vercel preview isn't working right now so just ignore me if the styling is okay)

Reply:
Good point! Making that change now.